src/databricks/labs/remorph/helpers/db_workspace_utils.py

```diff
@@ -0,0 +1,96 @@
+import logging
+
+from databricks.labs.blueprint.tui import Prompts
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors.platform import ResourceDoesNotExist
+
+logger = logging.getLogger(__name__)
+
+
+class DatabricksSecretsClient:
```

Comment:
move all of the methods from here to `ReconConfigPrompts`. You don't need to create secret scope client over an existing SDK client :)

Reply:
Done, refactored the code